package/ci/android-x86.sh

```diff
@@ -140,6 +140,7 @@ cmake .. \
     -DWITH_STBVORBISAUDIOIMPORTER=OFF \
     -DWITH_STLIMPORTER=ON \
     -DWITH_TINYGLTFIMPORTER=ON \
+    -DWITH_WEBPIMPORTER=ON \
```

Comment:
Getting libwebp on Android is a task that could easily take weeks, so don't even bother for now :) It can always be done later. Keep the option though but set it to `OFF`, to make it clear that it was explicitly disabled instead of forgotten:

```suggestion
    -DWITH_WEBPIMPORTER=OFF \
```

Same goes for `emscripten.sh` and `ios-simulator.sh`, those platforms don't have any easily available package either -- just explicitly disable it there, and if you added a package download for these three platforms in `circleci.yml`, remove it again. 

Building on Windows can also wait for later.

Reply:
I am happy that tests run successfully at least on Linux and macos. They were the easiest to configure with a bit of googling and looking how the configuration was done with other plugins as @Squareys suggested. The problem in the beginning was my fork on magnum-plugins not being updated. I updated my local repository by fetching latest changes and rebasing. Then I simply added libwebp/webp to the config.yml for installation on Linux and macos, and enabled the build of the WebPImporter.

Reply:
In my latest commit I removed all changes I made on android, ios, emscripten and appveyor configuration and set the build option to `OFF`.